cpp/ql/lib/semmle/code/cpp/ir/implementation/raw/internal/IRConstruction.qll

```diff
@@ -34,6 +34,9 @@ module Raw {
 
   cached
   predicate functionHasIR(Function func) { exists(getTranslatedFunction(func)) }
+  
+  cached
+  predicate varHasIRFunc(GlobalOrNamespaceVariable var) { any() } // TODO: restrict?
```

Comment:
TODO?

Also, does this cover static data members?

Reply:
No, it doesn't. I think it could be expanded to `StaticStorageDurationVariable`, but that might make the `getEnclosingFunction` issues worse...